server/src/main/scala/controllers/Application.scala

```diff
@@ -772,96 +795,102 @@ class Application @Inject()(implicit val config: Configuration,
   def exportFlightsWithSplitsAtPointInTimeCSV(pointInTime: String,
```

Comment:
should we rename this end point if it specifically contains API data?

Reply:
I'm actually not sure if there is any point in keeping this endpoint, it essentially does exactly the same thing as the other export, but with a slightly nicer url.

Reply:
I'll rename it for now, I think the idea was that this would be the endpoint we give to port operators.